api/src/testIntegration/java/bisq/api/http/UserEndpointIT.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.api.http;
+
+import bisq.api.http.model.ChangePassword;
+
+import bisq.common.util.Base64;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.isEmptyString;
+
+
+
+import com.github.javafaker.Faker;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.Container;
+import org.arquillian.cube.docker.impl.client.containerobject.dsl.DockerContainer;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+
+@RunWith(Arquillian.class)
+public class UserEndpointIT {
+
+    private static String validPassword = new Faker().internet().password();
+    private static String invalidPassword = getRandomPasswordDifferentThan(validPassword);
+    @DockerContainer
+    Container alice = ContainerFactory.createApiContainer("alice", "8081->8080", 3333, false, false);
+
+    private static String getRandomPasswordDifferentThan(String otherPassword) {
```

Comment:
What about moving this method down, below public tests ?

Reply:
I had mixed feelings about it cause it's static.

Reply:
Does it need to be static ? Also `invalidPassword` and `validPassword` fields are static, is there reason for that ?